tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -314,6 +319,56 @@ public void testConsumerListOffsetLatest() throws Exception {
         assertEquals(listOffsetResponse.responseData().get(tp).timestamp, Long.valueOf(0));
     }
 
+    /**
+     * Test the sending speed of fetch request when the readable data is less than fetch.minBytes.
+     */
+    @Test(timeOut = 60000)
+    public void testFetchMinBytes() throws Exception {
+        String topicName = "testMinBytesTopic";
+        TopicPartition tp = new TopicPartition(topicName, 0);
+
+        // create partitioned topic.
+        admin.topics().createPartitionedTopic(topicName, 1);
+        List<TopicPartition> topicPartitions = new ArrayList<>();
+        topicPartitions.add(tp);
+
+        int maxResponseBytes = 800;
+        int maxPartitionBytes = 900;
+        int maxWaitMs = 3000;
+        //case1: consuming an empty topic
+        KafkaHeaderAndRequest fetchRequest1 = createFetchRequest(maxWaitMs,
+                1,
+                maxResponseBytes,
+                maxPartitionBytes,
+                topicPartitions,
+                Collections.EMPTY_MAP);
+        CompletableFuture<AbstractResponse> responseFuture1 = new CompletableFuture<>();
+        Long startTime1 = System.currentTimeMillis();
+        kafkaRequestHandler.handleFetchRequest(fetchRequest1, responseFuture1);
+        responseFuture1.get();
+        Long endTime1 = System.currentTimeMillis();
+        log.info("cost time1:" + (endTime1 - startTime1));
+
+        //case2: consuming an  topic after produceing data
+        KafkaProducer<String, String> kProducer = createKafkaProducer();
+        produceData(kProducer, topicPartitions, 10);
+
+        KafkaConsumer<String, String> consumer = createKafkaConsumer();
+        consumer.assign(topicPartitions);
+        consumer.seekToBeginning(topicPartitions);
+        Long startTime2 = System.currentTimeMillis();
+        consumer.poll(Duration.ofMillis(maxWaitMs));
+        Long endTime2 = System.currentTimeMillis();
+        log.info("cost time2:" + (endTime2 - startTime2));
```

Comment:
Case 1 is creating a FetchRequest and call `handleFetchRequest` for it. However, case 2 is creating a Kafka consumer and call `poll`. I think we should also use a consumer for case 1, otherwise the comparison will be meaningless.

Reply:
You can specify `ConsumerConfig.FETCH_MAX_WAIT_MS_CONFIG` config for `maxWaitMs` like

```java
        props.put(ConsumerConfig.FETCH_MAX_WAIT_MS_CONFIG, 3000);
```